src/content/docs/en/tutorial/2-pages/index.mdx

```diff
@@ -14,16 +14,9 @@ import Box from '~/components/tutorial/Box.astro';
 
 Now that you have a working site on the web, it's time to add pages and posts!
 
-## Where are you now?
+## Looking ahead
 
-You can find the code at this point in the tutorial on [GitHub](https://github.com/withastro/blog-tutorial-demo/tree/unit-2/start) or
-[StackBlitz](https://stackblitz.com/github/withastro/blog-tutorial-demo/tree/unit-2/start?file=src/pages/index.astro).
-
-Check your code against this example, or if you've just joined us, fork on StackBlitz to start coding along in your browser from here.
-
-## Where are you going?
-
-In this unit, you'll
+In this unit, you'll:
```

Comment:
I know it's what it's there already, but not sure reducing "you will" to "you'll" is the best...

Reply:
I actually changed this too at first, but a) other sections do use "you'll" liberally. (though you're right, I don't like it in such a short sentence) and b) thought it might be easier on the translators not to do too much editing here.  😆

Reply:
Yeah, I do think most languages wouldn't actually change the sentence, it's just a shortening and it's merged with other changes. It'd be *way* worse if it was just that change alone.

I'd change it imho, but it's not a big issue at all.

Reply:
And you did while I was typing lol :)